src/db/base/base-kuzu-client.ts

```
@@ -0,0 +1,126 @@
+
+/**
+ * Base class for Kuzu database operations
+ * Provides common configuration, logging, and utility methods
+ */
+export abstract class BaseKuzuClient {
+  public readonly dbPath: string;
+  protected logger = kuzuLogger.child({ component: 'BaseKuzuClient' });
+
+  constructor(clientProjectRoot: string) {
+    this.dbPath = this.initializeDbPath(clientProjectRoot);
+    this.logger = kuzuLogger.child({ 
```

ESLint

[error] Delete `·`

(prettier/prettier)